browser/ui/webui/brave_vpn/vpn_panel_handler.cc

```diff
@@ -38,6 +40,21 @@ bool ShouldOpenSingletonTab(const std::string& type) {
   return type == "manage" || type == "privacy" || type == "about";
 }
 
+void ShowSingletonVPNTab(Browser* browser, const GURL& url) {
+  for (auto i = 0; i < browser->tab_strip_model()->GetTabCount(); i++) {
+    auto* web_contents = browser->tab_strip_model()->GetWebContentsAt(i);
+    const GURL& contents_url = web_contents->GetVisibleURL();
+    bool is_equal = contents_url.SchemeIs(url.scheme()) &&
```

Comment:
Curious `contents_url == gurl` doesn't work?

Reply:
https://github.com/brave/brave-core/pull/14486#discussion_r939451726

Reply:
also we do not care about query/ref parameters

Reply:
Got it. Thanks!